ros/src/driveworks/packages/autoware_driveworks_gmsl_interface/CMakeLists.txt

```diff
@@ -3,16 +3,7 @@ project(autoware_driveworks_gmsl_interface)
 
 find_package(CUDA)
 
-if ("${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "aarch64" AND CUDA_FOUND)
-    if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
-      set(DRIVEWORKS_LINKER_FLAGS "-Wl,--unresolved-symbols=ignore-in-object-files -Wl,--allow-shlib-undefined")
-    elseif(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
-      set(DRIVEWORKS_LINKER_FLAGS "-Wl,-undefined,dynamic_lookup")
-    endif()
-
-    add_definitions(-DVIBRANTE)
-
-    find_package(catkin REQUIRED COMPONENTS
+find_package(catkin REQUIRED COMPONENTS
```

Comment:
We can just use these two lines and it'll work:

```
     find_package(catkin REQUIRED)
     catkin_package()
```

That way we don't have to find the message packages.

Reply:
@esteve Thanks for checking this. Is it ok to remove the definition?
I don't have a PX nor multiple gmsl cameras to test.
Can you confirm it is safe?

Reply:
Sorry again, but I don't follow the suggestion.
So you say we add 
```
find_package(catkin REQUIRED)
catkin_package()
```
with no packages at the beggining.

And, then again, `find_package` and `catkin_package` with the corresponding packages inside the `if`?

Reply:
Sorry I wasn't clearer. I meant adding those two lines after the `else()` statement in https://github.com/CPFL/Autoware/blob/develop/ros/src/driveworks/packages/autoware_driveworks_gmsl_interface/CMakeLists.txt#L93